truss/truss_handle.py

```diff
@@ -1090,7 +1099,17 @@ def wait_for_truss(
     except RetryError as retry_err:
         retry_err.reraise()
     if wait_for_server_ready:
-        _wait_for_model_server(url)
+        if model_server_stop_retry_criteria:
+            _wait_for_model_server.retry_with(
```

Comment:
instead of duplicating the decorator, can you do something like modify `_wait_for_model_server` to be this:

```
def _wait_for_model_server(url: str, wait_seconds: int = 120) -> Response:
    for attempt in Retrying(...):
        with attempt:
            return requests.get(...)
```